instrumentation/external-annotations/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/extannotations/SayTracedHello.java

```diff
@@ -24,6 +24,7 @@ public String newrelic() {
     return "hello!";
   }
 
+  // End of life -- never use this annotation in new development
```

Comment:
I added it out of an abundance of caution. 

I'd hate for a reader of this code to leave thinking that because this annotation is supported by otel that they could or should use them in new code. The comment costs nothing and explains that it's not supported. It's just test code, so feel free to delete it.

Reply:
I see @laurit's point, a comment on only this one annotation could leave readers thinking that we do recommend using the other annotations

I think all of these annotations are only targeted at legacy applications, while all new development should be using the OpenTelemetry annotations